sycl/test/basic_tests/info_ocl_version.cpp

```diff
@@ -0,0 +1,40 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: env SYCL_DEVICE_TYPE=HOST %t.out
+// RUN: env SYCL_BE=PI_OPENCL %CPU_RUN_PLACEHOLDER %t.out
+// RUN: env SYCL_BE=PI_OPENCL %GPU_RUN_PLACEHOLDER %t.out
+// RUN: env SYCL_BE=PI_OPENCL %ACC_RUN_PLACEHOLDER %t.out
+// RUN: env SYCL_BE=PI_LEVEL_ZERO %CPU_RUN_PLACEHOLDER %t.out
+// RUN: env SYCL_BE=PI_LEVEL_ZERO %GPU_RUN_PLACEHOLDER %t.out
+// RUN: env SYCL_BE=PI_LEVEL_ZERO %ACC_RUN_PLACEHOLDER %t.out
```

Comment:
CPU RT does not exist in LEVEL_ZERO. 
BE iteration is done on LIT level. There is no need to iterate them in the test.
The test will also run on CUDA backend. 
```suggestion
// RUN: %CPU_RUN_PLACEHOLDER %t.out
// RUN: %GPU_RUN_PLACEHOLDER %t.out
// RUN: %ACC_RUN_PLACEHOLDER %t.out
```

Reply:
Ah, ok. I thought it will only run on one BE, not on all of them. Fixed.